.github/workflows/ci.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - emscripten
```

Comment:
This should be removed.

Reply:
While working on the PR it's helpful if I can trigger CI on my fork.